recipes-core/easysplash/files/0001-Remove-Cargo.toml-panic-abort-flag.patch

```diff
@@ -0,0 +1,22 @@
+From 74b42bc52f910607dfc63f11ecf36e65934cbfc7 Mon Sep 17 00:00:00 2001
+From: "Rodrigo M. Duarte" <rodrigo.duarte@ossystems.com.br>
+Date: Tue, 31 Jan 2023 10:40:04 -0300
+Subject: [PATCH] Remove Cargo.toml panic abort flag
```

Comment:
Send this change to the project itself so we avoid the patch.

Reply:
Fixed at source: 
https://github.com/OSSystems/EasySplash
81a44392efe15fd12ea673f1c828a52e95103eea